lib/v2/magazinelib/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/:latest-magazine/:query?', require('./latest-magazine'));
```

Comment:
```suggestion
    router.get('/latest-magazine/:query?', require('./latest-magazine'));
```
I don't see the variable `ctx.params['latest-magazine']` being used at all